--- htaccess	2023-11-01 13:42:35
+++ htaccess2	2024-02-07 21:03:52
@@ -59,7 +59,8 @@
 # Various rewrite rules.
 <IfModule mod_rewrite.c>
   RewriteEngine on
-
+  RewriteCond %{REQUEST_FILENAME} !-f
+  RewriteRule ^files/(.*)$ https://gdc-public-files.s3.amazonaws.com/prod/$1 [L,R=301]
   # Set "protossl" to "s" if we were accessed via https://.  This is used later
   # if you enable "www." stripping or enforcement, in order to ensure that
   # you don't bounce between http and https.
