--- .htaccess	2024-02-13 22:15:33.805551864 +0000
+++ htaccess	2024-02-13 22:30:01.977965666 +0000
@@ -59,7 +59,9 @@
 # Various rewrite rules.
 <IfModule mod_rewrite.c>
   RewriteEngine on
-
+  RewriteEngine on
+  RewriteCond %{REQUEST_FILENAME} !-f
+  RewriteRule ^files/(.+)$ https://gdc-public-files.s3.amazonaws.com/prod/$1 [L,R=301]
   # Set "protossl" to "s" if we were accessed via https://.  This is used later
   # if you enable "www." stripping or enforcement, in order to ensure that
   # you don't bounce between http and https.
