--- .htaccess   2024-02-17 02:52:33.000000000 +0000
+++ web/.htaccess       2024-03-01 16:26:06.000000000 +0000
@@ -59,7 +59,9 @@
 # Various rewrite rules.
 <IfModule mod_rewrite.c>
   RewriteEngine on
-
+  RewriteCond %{REQUEST_FILENAME} !-f
+  RewriteRule ^sites/default/files/(.+)$ /system/files/$1%1 [L,R=301]
+  RewriteRule ^files/(.+)$ /system/files/$1 [L,R=301]
   # Set "protossl" to "s" if we were accessed via https://.  This is used later
   # if you enable "www." stripping or enforcement, in order to ensure that
   # you don't bounce between http and https.
